kernels/optimized/cpu/op_bmm.cpp

```diff
@@ -150,15 +150,14 @@ Tensor& opt_bmm_out(
   ET_KERNEL_CHECK(
       ctx, check_bmm_out_args(self, mat2, out), InvalidArgument, out);
 
-  constexpr auto name = "bmm.out";
```

Comment:
I don't suppose we are lucky enough that `static constexpr auto name = "bmm.out";` would fix it?